overlord/snapstate/handlers.go

```diff
@@ -275,7 +276,8 @@ func (m *SnapManager) installPrereqs(t *state.Task, base string, prereq []string
 		var err error
 		var ts *state.TaskSet
 		timings.Run(tm, "install-prereq", fmt.Sprintf("install %q", prereqName), func(timings.Measurer) {
-			ts, err = m.installOneBaseOrRequired(st, prereqName, defaultPrereqSnapsChannel(), onInFlightErr, userID)
+			noBaseTypeCheck := false
```

Comment:
s/BaseType/TypeBase/ ?

Reply:
Ah, I'm sorry, I missed that one before, fixed. Thank you.